server/service/src/purchase_order/update/validate.rs

```diff
@@ -74,7 +79,7 @@ pub fn validate(
         }
     }
 
-    Ok(purchase_order)
+    Ok((purchase_order, None))
```

Comment:
Just wondering if we need to add Status as a return option here? Seems like it's just none always? I could be missing context